flint.ui/package.json

```diff
@@ -1,34 +1,43 @@
 {
   "name": "flint-ui",
   "version": "1.1.0",
-  "description": "This project aims to implement a user-friendly UI for FLINT and GCBM runs",
   "private": true,
-  "license": "MIT",
+  "description": "This project aims to implement a user-friendly UI for FLINT and GCBM runs",
   "scripts": {
     "serve": "vue-cli-service serve --open",
-    "dev": "npm run serve",
     "build": "vue-cli-service build",
+    "build-watch": "vue-cli-service build-watch",
```

Comment:
Is there a difference here between colons and dashes? It looks like - is used in package name and : is used to tag scripts.